---
title: '<span style="display:none">**Karen**</span>'
output: 
  html_document:
    includes:
      after_body: footer.html
    css: figures.css
---

<script src="https://kit.fontawesome.com/0af1a424a5.js" crossorigin="anonymous"></script>

<style>
img { 
  width: 30%;
  margin: 0.6em ;
  border: 0.4em solid;
}

#myBtn {
  display: none; /* Hidden by default */
  position: fixed; /* Fixed/sticky position */
  bottom: 20px; /* Place the button at the bottom of the page */
  right: 30px; /* Place the button 30px from the right */
  z-index: 99; /* Make sure it does not overlap */
  border: none; /* Remove borders */
  outline: none; /* Remove outline */
  background-color: red; /* Set a background color */
  color: white; /* Text color */
  cursor: pointer; /* Add a mouse pointer on hover */
  padding: 15px; /* Some padding */
  border-radius: 10px; /* Rounded corners */
  font-size: 18px; /* Increase font size */
}

#myBtn:hover {
  background-color: #555; /* Add a dark-grey background on hover */
}
.main-container {
  width: 1800px;
  margin-left: auto;
  margin-right: auto;
}

h1, h2, h3, h4 {
text-align: center
}

.col-md-6 {
margin: 0px
padding: 0px
}

footer {
left: 50;
padding-left: 200px;
padding-right: 200px
width: 1000px;
background-color: white;
display: block;
text-align: center;
}


/* Responsive layout - makes the two columns/boxes stack on top of each other instead of next to each other, on small screens */
@media (max-width: 700px) {
  .main-container {
    width: 100%;
    height: auto;
  }
}

</style>


<div class = "row">

<div class="col-md-4" style="position: -webkit-sticky; position: sticky; top: 5%">

<div class="row px-0">
<style> img {width: 100%; text-align: center}</style>
![](images/karen.png)
<p  align="center">**Karen**</p>

<div class="col-md-5 mx-0">

<p  align="center"> If you want to learn about my experience, visit my [webpage](https://KarenGoncalves.github.io/Karen/experience.html).<br>I’d love to hear from you, so feel free to reach out:</p>

<p  align="center"><a href="mailto:karen.cristine.goncalves.dos.santos@uqtr.ca"  style="color:#fc1c03"><i class="fas fa-envelope"></i></a> <a href="mailto:cris.kgs@gmail.com" style="color:#03b6fc"><i class="fas fa-envelope"></i></a></p>


<p  align="center">Or connect with me on:</p>
<span style="font-size: 2em"> <p  align="center"> [<i class="fab fa-twitter"></i>](http://twitter.com/KarenCristineG1) [<i class="fab fa-linkedin-in"></i>](https://www.linkedin.com/in/karen-cristine-gon%C3%A7alves-dos-santos-05847a113/) [<i class="fab fa-github"></i>](http://github.com/KarenGoncalves) </p> </span>
</div>

<div class="col-md-7 mx-0" style="text-align: left">

  - [Research](#research)
  - [Skills](#skills)
    - [Soft skills](#soft)
    - [Technical](#technical)
    - [Languages](#languages)
  - [Publications](#pubs)
    - [Papers](#papers)
    - [Oral presentations](#presentations)
    - [Posters](#posters)

</div>
</div>
</div>

<div class = "col-md-8">


# **Research path** {#research}

<div style="width: 100%; display: block">


I have a bachelor degree in Biology from Brazil at [Unila](https://portal.unila.edu.br/). There I researched the interaction of two bacteria, _Gluconacetobacter diazotrophicus_ and _Azospirillum brasiliensis_, and a fungi, _Trichoderma harzianum_, with the Mate plant, _Ilex paraguariensis_, under the direction of [Cristian A. Rojas](https://www.researchgate.net/profile/Cristian_Rojas6) and [Cleto Peres](https://www.researchgate.net/profile/Cleto_Peres).

I am currently doing my PhD in Quebec with [Hugo Germain](http://www.germainhugo.com/) and [Isabel Desgagne-Penix](https://oraprdnt.uqtr.uquebec.ca/pls/public/gscw031?owa_no_site=3466) at [UQTR](http:/uqtr.ca) on **_Transcriptomics and metabolomics of Arabidopsis in response to fungal virulence proteins_**. Although my PhD program is in Cellular and molecular biology, I did all the analyses of my transcriptomic and metabolomic data. I also developed a R package for selection of reference genes from transcriptomic data (DOI: https://doi.org/10.1186/s12864-019-6426-2), you can access it on  [Github](https://github.com/KarenGoncalves/CustomSelection) too. I have collaborated in other projects throughout my PhD doing _de novo_ transcriptome assembly and annotation

In 2018, I did an internship at [Peter Solomon's lab](https://biology.anu.edu.au/people/academics/peter-solomon) with [Megan McDonald](https://biology.anu.edu.au/people/academics/megan-mcdonald), where I used MinIon to sequence the genome of several isolates of the fungus _Bipolaris sorokiniana_, basecalled the reads and assembled them and finally aligned the new genomes to a reference to study structural changes in the chromosomes.

Finally, I am secretary of the Centre SEVE's International Partnership Committee (CPICS, 2018-present). In this role, that I share with two other people, I am responsible for the organization of documents of the committee, for the transcription of meetings and translation of documents. I also designed the [CPICS website](https://cpics.netlify.app) and the website for this company (MaKaCo).

</div>

------------------------------------

# **Skills** {#skills}


## Soft skills {#soft}

  - Fast learner
  - Critical thinking
  - Creative
  - Mentoring
  - Good written and oral communication
  - Willing to accept feedback
  - Willingness to learn
<br><br>
  
## Technical {#technical}

<table style="margin: 0px;width: 100%;">
<tr>
<td>- MS Office </td><td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td> <td class="righty"><i  class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i> </td></tr>
<tr>
<td>- R and Bioconductor </td><td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td> <td class="righty"><i  class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="far fa-circle"></i></span> </td></tr>
<tr>
<td>- Unix </td><td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td> <td class="righty"><i  class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="far fa-circle"></i></td></tr>
<tr>
<td>- High performance computing </td><td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td> <td class="righty"><i  class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="far fa-circle"></i><i class="far fa-circle"></i> </td></tr>
<tr>
<td>- Markdown </td><td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td> <td class="righty"><i  class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="far fa-circle"></i><i class="far fa-circle"></i> </td></tr>
<tr>
<td>- HTML </td><td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td> <td class="righty"><i  class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="far fa-circle"></i><i class="far fa-circle"></i> </td></tr>
<tr>
<td>- CSS </td><td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td> <td class="righty"><i  class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="far fa-circle"></i><i class="far fa-circle"></i> </td></tr>
<tr>
<td>- SQL </td><td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td> <td class="righty"><i  class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="far fa-circle"></i><i class="far fa-circle"></i><i class="far fa-circle"></i> </td></tr>
<tr>
<td>- Python </td><td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td> <td class="righty"><i  class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="fas fa-circle"></i><i class="far fa-circle"></i><i class="far fa-circle"></i><i class="far fa-circle"></i></span>  </td></tr>
</table>
  
<br><br>
    
## Languages {#languages}

<div style="display:inline; float: left; width: 100%;">
| Language | Read | Write | Speak | Understand | Native |
|:----|:---:|:---:|:---:|:---:|:---:|
| English | Fluent | Advanced | Fluent | Fluent | |
| French | Fluent | Intermediary | Advanced | Fluent | |
| Portuguese | Fluent | Fluent | Fluent | Fluent | Yes |
| Spanish  | Fluent | Fluent | Advanced | Fluent | |
</div>

------------------------------------

# **Publications** {#pubs}

## **Papers** {#papers}

### *2020*

**dos Santos, K.C.G.**, Pelletier, G., Séguin, A. Guillemette, F., Hawkes, J., Desgagné-Penix, I. & Germain, H. Differential alteration of plant functions by homologous fungal candidate effectors. bioRxiv (2020) DOI: [10.1101/2020.10.30/363010](https://doi.org/10.1101/2020.10.30.363010)

**dos Santos, K.C.G.**, Desgagné-Penix, I. & Germain, H. Custom selected reference genes outperform pre-defined reference genes in transcriptomic analysis. BMC Genomics 21, 35 (2020) DOI: [10.1186/s12864-019-6426-2](https://doi.org/10.1186/s12864-019-6426-2)

### *2019*

Fradj, N., **dos Santos, K.C.G.**, Montigny, N., Awwad, F., Boumghar, Y., Germain, H., Desgagné-Penix, I.  2019. RNA-Seq de Novo Assembly and Differential Transcriptome Analysis of Chaga (_Inonotus obliquus_) Cultured with Different Betulin Sources and the Regulation of Genes Involved in Terpenoid Biosynthesis. International Journal of Molecular Sciences 20(18), 4334. DOI: [10.3390/ijms20184334](https://doi.org/10.3390/ijms20184334)

Lorrain, C., **Gonçalves dos Santos, K.C.**, Germain, H., Hecker, A. and Duplessis, S. 2019. Advances in understanding obligate biotrophy in rust fungi. New Phytologist, 222: 1190-1206. DOI: [10.1111/nph.15641](https://doi.org/10.1111/nph.15641)

### *2018*

**dos Santos, K.C.G.**, McDonald, M. 2018. High molecular weight DNA extraction for long read sequencing v1. Protocols.io, DOI: [10.17504/protocols.io.uppevmn](https://doi.org/10.17504/protocols.io.uppevmn)

Ahmed, M.B., **Santos, K.C.G.d.**, Sanchez, I.B. et al. 2018. A rust fungal effector binds plant DNA and modulates transcription. Scientific Reports 8, 14718, DOI: [10.1038/s41598-018-32825-0](https://doi.org/10.1038/s41598-018-32825-0)

### *2013*

**dos Santos, K.C.G.**; Lopez, D.S.H. ; Perez, L.M.; Roberti, L.; Schmid, P.; Figueiredo, I.; Peres, C.K.; C.A. Rojas. 2013. _Gluconacetobacter diazotrophicus_ enhances growth initial stages of _Ilex paraguariensis_ development. II Conferência Iberoamericana de Interaçoes Benéficas Microorganismos-Planta-Ambiente (IBEMPA), p. 450-451 (Extended abstract).



## **Oral presentations** {#presentations}


### *2020*

**dos Santos, K.C.G.**, Desgagné-Penix, I. & Germain, H. 2017. Impact of fungal virulence proteins on in plants. Conferences Webs du Centre SÉVE (online).

### *2017*

**dos Santos, K.C.G.**, Desgagné-Penix, I. & Germain, H. 2017. Transcriptomic and metabolomic analysis of the alteration of physiological processes in a collection of transgenic plants expressing fungal effectors. Centre SÉVE, Orford, Canada

### *2014*

**dos Santos, K.C.G.**; Lopez, D.S.H. ; Perez, L.M.; Roberti, L.; Schmid, P.; Figueiredo, I.; Peres, C.K.; Rojas, C.A. 2014. Análise da interação endofítica de microorganismos benéficos e a erva-mate. **III Encontro de Iniciação Científica da Unila “Pesquisa no século XXI: desafios e possibilidades”**. In: [*Anais do III Encontro de Iniciação Científica da Unila “Pesquisa no século XXI: desafios e possibilidades”*](http://dspace.unila.edu.br/123456789/736), p. 135, Foz do Iguaçu, Brazil.

### *2013*

**dos Santos, K.C.G.**; Lopez, D.S.H. ; Perez, L.M.; Roberti, L.; Schmid, P.; Figueiredo, I.; Peres, C.K.; Rojas, C.A.  2013. Evaluación de la respuesta de la yerba mate a la inoculación con microorganismos potencialmente benéficos. II Congreso Paraguayo de Recursos Naturales y Medio Ambiente, Asunción, Paraguay. 

**dos Santos, K.C.G.**; Lopez, D.S.H. ; Perez, L.M.; Roberti, L.; Schmid, P.; Figueiredo, I.; Peres, C.K.; Rojas, C.A. 2013. Avaliação da Resposta da Erva-mate à Inoculação com Microorganismos Potencialmetne Benéficos. **II Encontro de Iniciação Científica da Unila "Resultados em debate"**. In: [*Anais do 2º Encontro de Iniciação Científica da Unila "Resultados em debate"*](http://dspace.unila.edu.br/123456789/735), p. 55-56, Foz do Iguaçu, Brazil.



## **Poster presentations** {#posters}

### *2020*

**dos Santos, K.C.G.**, Pelletier, G., Séguin, A. Guillemette, F., Hawkes, J., Desgagné-Penix, I. & Germain, H. Differential alteration of plant functions by homologous fungal candidate effectors. [Concours d'affiches scientifiques étudiantes 2020 du Centre SÈVE](https://event.fourwaves.com/fr/affichesetudiantescs/abstracts/24a206e5-03ec-43ce-b763-1a920c3f9736) (online).

**dos Santos, K.C.G.**, Desgagné-Penix, I. & Germain, H. 2020. Étude des plantes exprimant des protéines de virulence de la rouille du peuplier. Contest of scientific Posters - UQTR, Trois-Rivières, Canada.

### *2019*

**dos Santos, K.C.G.**, Desgagné-Penix, I. & Germain, H. 2019. Transcriptomics and metabolomics of Arabidopsis expressing effectors from the poplar rust. Réunion conjointe Centre SÉVE-SPPQ, Bromont, Canada.

**dos Santos, K.C.G.**, Desgagné-Penix, I. & Germain, H. 2019. Effectors from poplar rust modulate Arabidopsis transcriptome. IS-MPMI 2019, Glasgow, United Kingdom.

**dos Santos, K.C.G.**, Desgagné-Penix, I. & Germain, H. 2019. Custom selected reference genes outperform pre-defined reference genes in transcriptomic analysis. Plant Canada 2019, Guelph, Canada.

**dos Santos, K.C.G.**, Desgagné-Penix, I. & Germain, H. 2019. Custom selected reference genes outperform pre-defined reference genes in transcriptomic analysis. Plant Canada 2019, Guelph, Canada.

**dos Santos, K.C.G.**, Desgagné-Penix, I. & Germain, H. 2019. Facteurs de virulence fongique modulent le transcriptome d'Arabidopsis. Contest of scientific Posters - UQTR, Trois-Rivières, Canada.

### *2018*

**dos Santos, K.C.G.**, Desgagné-Penix, I. & Germain, H. 2018. Analysis of transcriptomic alterations in a collection of transgenic plants expressing fungal effectors. Plant Biology 2018, Montreal, Canada.

**dos Santos, K.C.G.**, Desgagné-Penix, I. & Germain, H. 2018. Analysis of transcriptomic alterations in a collection of transgenic plants expressing fungal effectors. Contest of scientific Posters - UQTR, Trois-Rivières, Canada.

### *2017*

**dos Santos, K.C.G.**, Desgagné-Penix, I. & Germain, H. 2017. Transcriptomic and metabolomic analysis of the alteration of physiological processes in a collection of transgenic plants expressing fungal effectors. Contest of scientific posters - UQTR, Trois-Rivières, Canada.

### *2014*

**dos Santos, K.C.G.**; Lopez, D.S.H. ; Perez, L.M.; Roberti, L.; Schmid, P.; Figueiredo, I.; Peres, C.K.; Rojas, C.A. 2014. Promoción del crecimiento de *Ilex paraguariensis* mediada por *Gluconacetobacter diazotrophicus* y su relación con el metabolismo del nitrógeno. In: [**Memorias del VI Congreso Sudamericano de Yerba Mate y II Simposio Internacional de Yerba-Mate y Salud**](http://www.congresos-rohr.com/yerbamate2014/trabajo_docs/65_T74_pdfwxD7X_pdf3LgCf.pdf), Montevideo, Uruguay.

**dos Santos, K.C.G.**; Lopez, D.S.H. ; Perez, L.M.; Roberti, L.; Schmid, P.; Figueiredo, I.; Peres, C.K.; C.A. Rojas. 2014. Análise da interação endofítica de microorganismos benéficos e a erva-mate. **III Encontro de Iniciação Científica da Unila “Pesquisa no século XXI: desafios e possibilidades”**, Foz do Iguaçu, Brazil.

### *2013*

**dos Santos, K.C.G.**; Lopez, D.S.H. ; Perez, L.M.; Roberti, L.; Schmid, P.; Figueiredo, I.; Peres, C.K.; Rojas, C.A. 2013. Evaluación de la respuesta de la yerba mate a la inoculación con microorganismos potencialmente benéficos. **IV Congreso de Agrobiotecnología, Propiedad Intelectual y Políticas Públicas**, Posadas, Argentina.

**dos Santos, K.C.G.**; Lopez, D.S.H. ; Perez, L.M.; Roberti, L.; Schmid, P.; Figueiredo, I.; Peres, C.K.; Rojas, C.A. 2013. Avaliação da Resposta da Erva-mate à Inoculação com Microorganismos Potencialmetne Benéficos. **II Encontro de Iniciação Científica da Unila "Resultados em debate"**, Foz do Iguaçu, Brazil.



</div>

</div>
<!--
<button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>
<script>
//Get the button
var mybutton = document.getElementById("myBtn");

// When the user scrolls down 20px from the top of the document, show the button
window.onscroll = function() {scrollFunction()};

function scrollFunction() {
  if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
    mybutton.style.display = "block";
  } else {
    mybutton.style.display = "none";
  }
}

// When the user clicks on the button, scroll to the top of the document
function topFunction() {
  document.body.scrollTop = 0;
  document.documentElement.scrollTop = 0;
}
</script>-->
